doc/introduction/compiling_circuits.rst

```diff
@@ -15,6 +15,73 @@ or replace a large circuit by a number of smaller circuits.
 Compilation functionality is mostly designed as **transforms**, which you can read up on in the
 section on :doc:`inspecting circuits </introduction/inspecting_circuits>`.
 
+Simplifying Operators
+----------------------
+
+PennyLane provides the :func:`~.pennylane.simplify` function to simplify single operators, quantum
+functions, QNodes and tapes. This function has several purposes:
+
+* Reducing the arithmetic depth of the given operators to its minimum.
+* Grouping like terms in sums and products.
+* Resolving products of Pauli operators.
+* Combining identical rotation gates by summing its angles.
+
+Here are some simple simplification routines:
+
+>>> qml.simplify(qml.adjoint(qml.RX(1.23, 0)))
+RX(11.336370614359172, wires=[0])
+>>> qml.simplify(qml.ops.Pow(qml.RX(1, 0), 3))
+RX(3.0, wires=[0])
+>>> qml.simplify(qml.op_sum(qml.PauliY(3), qml.PauliY(3)))
+2*(PauliY(wires=[3]))
+>>> qml.simplify(qml.RX(1, 0) @ qml.RX(1, 0))
+RX(2.0, wires=[0])
+>>> qml.simplify(qml.prod(qml.PauliX(0), qml.PauliZ(0)))
+-1j*(PauliY(wires=[0]))
+
+Now lets simplify a nested operator:
+
+>>> nested_op = qml.prod(qml.prod(qml.PauliX(0), qml.op_sum(qml.RX(1, 0), qml.PauliX(0))), qml.RX(1, 0))
+>>> qml.simplify(nested_op)
+(PauliX(wires=[0]) @ RX(2.0, wires=[0])) + RX(1.0, wires=[0])
+
+Several simplifications steps are happening here. First of all, the nested products are removed:
+.. code-block:: python
+    qml.prod(qml.PauliX(0), qml.op_sum(qml.RX(1, 0), qml.PauliX(0)), qml.RX(1, 0))
```

Comment:
```suggestion

.. code-block:: python

    qml.prod(qml.PauliX(0), qml.op_sum(qml.RX(1, 0), qml.PauliX(0)), qml.RX(1, 0))
```

Sphinx can be really finicky sometimes.  I think this will do the trick.